splipy/SplineModel.py

```diff
@@ -724,14 +726,15 @@ def add(self, obj, raise_on_twins=True):
         true.
 
         :param SplineObject obj: The object to add
-        :param bool raise_on_twins: If true, raise an error when
-            'twins' are detected, i.e. two patches that share
-            identical nodes of lower parametric dimension but which
-            are different. For example, two surfaces with identical
-            edges but different interior, like a 'pillow'. If false,
-            such cases will be considered two genuinely different
-            patches. Setting this to true (default) allows catching a
-            number of typical topological problems.
+        :param bool raise_on_twins: Should be a list of parametric
+            dimensions where 'twins' are disallowed, i.e. two patches
+            that share identical nodes of lower parametric dimension
+            but which are different. For example, two surfaces with
+            identical edges but different interior, like a
+            'pillow'. If None, such cases will be considered two
```

Comment:
I don't understand the pillow analogy

Reply:
Yeah, I don't know how else to explain it briefly. :-P

Reply:
![20201202_142457](https://user-images.githubusercontent.com/619375/100878290-47e1d800-34aa-11eb-9cb0-7e81c7de164d.jpg)

I'm imagining a situation like this where one surface forms the top of the pillow and another forms the bottom. They have identical edges, so SplineModel will then try to match the interior, failing to do so.